MU/NewsModule/Controller/MessageController.php

```diff
@@ -248,4 +248,75 @@ public function handleSelectedEntriesAction(
     }
     
     // feel free to add your own controller methods here
+    /**
+     * Import news items from an old news site.
+     *
+     * @Route("/messages/importNewsArticles")
+     *
+     * @param Request $request Current request instance
+     *
+     *
+     * @throws RuntimeException Thrown if executing the workflow action fails
+     */
+
+    public function importNewsArticlesAction(Request $request){
+        //check permission, only admins can do this.
+        if (!$this->hasPermission( '::', '::', ACCESS_ADMIN)) {
```

Comment:
How about `if (!$this->hasPermission('MUNewsModule::', '::', ACCESS_ADMIN)) {`?

Reply:
Works for me